packages/alfa-rules/src/sia-er62/rule.ts

```diff
@@ -522,9 +522,18 @@ namespace Distinguishable {
       // Any difference in `background-image` is considered enough. If different
       // `background-image` ultimately yield the same background (e.g. the same
       // image at two different URLs), this creates false negatives.
+      // When there is no `background-image` set on the link, we consider it to be the same as the container's
       hasComputedStyle(
         "background-image",
-        not((image) => image.equals(imageReference)),
+        not((image) => {
+          let hasNoBackgroundImage: boolean = false;
+          for (const value of image.values) {
+            if (Keyword.isKeyword(value) && value.equals(Keyword.of("none"))) {
+              hasNoBackgroundImage = true;
+            }
+          }
```

Comment:
This is not super elegant, but it turns out when it comes to the `background-image` property, transparent means that the value of the css property is `none`.

Reply:
🤔 I feel we should merge both transparent tests. Something like `hasNoBackground = [color: transparent] and [image: none]` . That will make the logic more apparent and a bit safer.